src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -151,9 +163,17 @@ export default class CancerStudyTreeData {
                     );
                 }
 
-                let searchTerms = isCancerType
-                    ? node.name
-                    : [node.name, (node as CancerStudy).description].join('\n');
+                const foundStudyTags =
+                    allStudyTags.length &&
+                    isStudy(node) &&
+                    allStudyTags.find(
+                        t => t.studyId === (node as CancerStudy).studyId
+                    );
+                let studyTags = '';

```

Comment:
ternary would be clearer

Reply:
also, would be nice if we could do null or undefined instead of empty string

Reply:
Fixed in [2009473](https://github.com/cBioPortal/cbioportal-frontend/pull/4571/commits/2009473b6d81c094ba4246ee45947fd9d316e4fe)